Catalog doc (#95)
miamtech/mealz-documentation: docs/android/features/catalog/customize-view/toolbar.mdx

---
sidebar_position: 4
label: "Toolbar"
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

# Toolbar

> This template allow you to customize Categories ligne in home page of the catalog

Reviewer: ```suggestion
This template allows you to customize the toolbar at the top of the catalog page.
```

Author: ok
